frontend/app/layout.js

```diff
@@ -1,27 +1,32 @@
-import localFont from 'next/font/local';
-import './globals.css';
-import { SessionProvider } from 'next-auth/react';
+import localFont from "next/font/local";
+import "./_styles/globals.css";
+import { SessionProvider } from "next-auth/react";
 
 const geistSans = localFont({
-  src: './fonts/GeistVF.woff',
-  variable: '--font-geist-sans',
-  weight: '100 900',
+  src: "./fonts/GeistVF.woff",
+  variable: "--font-geist-sans",
+  weight: "100 900",
 });
 const geistMono = localFont({
-  src: './fonts/GeistMonoVF.woff',
-  variable: '--font-geist-mono',
-  weight: '100 900',
+  src: "./fonts/GeistMonoVF.woff",
+  variable: "--font-geist-mono",
+  weight: "100 900",
```

Comment:
I'm pretty sure in tailwind you can define your custom font in theme config and then use it in the classes on granular level like `text-sans` or `text-mono`. Is there any reason behind this particular approach?